oci builder fails with packer 1.8.4

**What steps did you take and what happened:**
`make build-oci-ubuntu-2004`

```
➜  capi git:(master) ✗ make build-oci-ubuntu-2004
hack/ensure-ansible.sh
Starting galaxy collection install process
Nothing to do. All requested collections are already installed. If you want to reinstall them, consider using `--force`.
hack/ensure-packer.sh
packer build -var-file="/home/builder/workspace/image-builder/images/capi/packer/config/kubernetes.json"  -var-file="/home/builder/workspace/image-builder/images/capi/packer/config/cni.json"  -var-file="/home/builder/workspace/image-builder/images/capi/packer/config/containerd.json"  -var-file="/home/builder/workspace/image-builder/images/capi/packer/config/ansible-args.json"  -var-file="/home/builder/workspace/image-builder/images/capi/packer/config/goss-args.json"  -var-file="/home/builder/workspace/image-builder/images/capi/packer/config/common.json"  -var-file="/home/builder/workspace/image-builder/images/capi/packer/config/additional_components.json"  -color=true -var-file="/home/builder/workspace/image-builder/images/capi/packer/oci/ubuntu-2004.json"  packer/oci/packer.json
Error: Failed to initialize build "oracle-oci"

error initializing builder 'oracle-oci': Unknown builder oracle-oci


make: *** [Makefile:470: build-oci-ubuntu-2004] Error 1
➜  capi git:(master) ✗ packer --version
1.8.4
➜  capi git:(master) ✗
```

**What did you expect to happen:**
OCI builds should be successful 


**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

https://github.com/hashicorp/packer/pull/11983

**Environment:**

Project ([Image Builder for Cluster API](https://github.com/kubernetes-sigs/image-builder/tree/master/images/capi), [kube-deploy/imagebuilder](https://github.com/kubernetes-sigs/image-builder/tree/master/images/kube-deploy/imagebuilder), [konfigadm](https://github.com/kubernetes-sigs/image-builder/tree/master/images/konfigadm)):

Additional info for Image Builder for Cluster API related issues:

- OS (e.g. from `/etc/os-release`, or `cmd /c ver`):
- Packer Version:
- Packer Provider:
- Ansible Version: 
- Cluster-api version (if using):
- Kubernetes version: (use `kubectl version`):

/kind bug
[One or more /area label. See https://github.com/kubernetes-sigs/cluster-api/labels?q=area for the list of labels]

--- a/images/capi/Makefile
+++ b/images/capi/Makefile
@@ -117,6 +117,7 @@ deps-oci: ## Installs/checks dependencies for OCI builds
 deps-oci:
 	hack/ensure-ansible.sh
 	hack/ensure-packer.sh
+	packer plugins install github.com/hashicorp/oracle
 
 .PHONY: deps-vbox
 deps-vbox: ## Installs/checks dependencies for VirtualBox builds

